package.json

```
@@ -9,10 +9,11 @@
     "dev": "ts-node-dev -r tsconfig-paths/register --respawn --transpile-only src/main.ts",
     "clean": "rm -rf dist bin node_modules",
     "codegen": "graphql-codegen-esm --config graphql.config.ts",
-    "test": "vitest",
-    "test:ui": "vitest --ui",
-    "test:run": "vitest run",
-    "test:coverage": "vitest run --coverage",
+    "e2e": "yarn test --project e2e",
```

This [ran](https://parsley.mongodb.com/evergreen/sage_ubuntu2204_small_e2e_patch_ae79666152e6526968d39c28214920603795edd5_688bc3b1dc28b700076b0422_25_07_31_19_27_46/0/task?bookmarks=0%2C700&selectedLineRange=L488&shareLine=488) `vitest --project unit --project e2e`

```suggestion
    "e2e": "vitest --project e2e",
```